sdk/src/sysvar/instructions.rs

```diff
@@ -0,0 +1,51 @@
+//! This account contains the serialized transaction instructions
+//!
+
+use crate::instruction::Instruction;
+use crate::sanitize::SanitizeError;
+use crate::sysvar::Sysvar;
+
+pub type Instructions = Vec<Instruction>;
+
+crate::declare_sysvar_id!("instructions1111111111111111111111111111111", Instructions);
```

Comment:
nits: if it's not too late, maybe follow the naming convention?: `Sysvar1nstructions1111111111111111111111111` (`Sysvar` prefix and camel case).


```
$ git grep -A1 declare_sysvar_id!
sdk/src/sysvar/clock.rs:crate::declare_sysvar_id!("SysvarC1ock11111111111111111111111111111111", Clock);
sdk/src/sysvar/clock.rs-
--
sdk/src/sysvar/epoch_schedule.rs:crate::declare_sysvar_id!("SysvarEpochSchedu1e111111111111111111111111", EpochSchedule);
sdk/src/sysvar/epoch_schedule.rs-
--
sdk/src/sysvar/fees.rs:crate::declare_sysvar_id!("SysvarFees111111111111111111111111111111111", Fees);
sdk/src/sysvar/fees.rs-
--
sdk/src/sysvar/recent_blockhashes.rs:declare_sysvar_id!(
sdk/src/sysvar/recent_blockhashes.rs-    "SysvarRecentB1ockHashes11111111111111111111",
--
sdk/src/sysvar/rent.rs:crate::declare_sysvar_id!("SysvarRent111111111111111111111111111111111", Rent);
sdk/src/sysvar/rent.rs-
--
sdk/src/sysvar/rewards.rs:crate::declare_sysvar_id!("SysvarRewards111111111111111111111111111111", Rewards);
sdk/src/sysvar/rewards.rs-
--
sdk/src/sysvar/slot_hashes.rs:crate::declare_sysvar_id!("SysvarS1otHashes111111111111111111111111111", SlotHashes);
sdk/src/sysvar/slot_hashes.rs-
--
sdk/src/sysvar/slot_history.rs:crate::declare_sysvar_id!("SysvarS1otHistory11111111111111111111111111", SlotHistory);
sdk/src/sysvar/slot_history.rs-
--
sdk/src/sysvar/stake_history.rs:crate::declare_sysvar_id!("SysvarStakeHistory1111111111111111111111111", StakeHistory);
sdk/src/sysvar/stake_history.rs-
```